### PR TITLE
Add recent event (fixes #27)

### LINKED ISSUE #27: Add recent event (Prague)
Videos: https://www.youtube.com/playlist?list=PLnXfazh66kVdd4tf0AqlfW4-VVXOHlj5N
Details on Meetup: https://www.meetup.com/frontendisti/events/246444623/

**Slides:**

- Iakov: https://docs.google.com/presentation/d/1b_V9H5vMOb2SkXh4Ev53hEADMlkM6FUL7HQriVtpbXI/edit#slide=id.p
- Martin: https://www.slideshare.net/machal/metriky-rychlosti-natn
- Láďa: https://www.slideshare.net/ladislavprskavec/how-you-can-kill-wordpress

### PR DESCRIPTION
**Pull request by dlouhak**
#27

### CODE REVIEW

File: src/data/events.yml [hunk 1]
```diff
@@ -986,3 +986,34 @@
...
+            name_link: https://twitter.com/IakovMarkov
+            company: Ataccama
+            company_link: https://www.ataccama.com/
+            title: Chrome&nbsp;DevTools, základní i&nbsp;pokročilé měření výkonu
+            youtube_id: 8DCQAJWwhFE
+            image: iakov-markov.jpg
+        -   name: Martin Michálek
+            name_link: https://twitter.com/machal
+            company: Freelancer
+            company_link: ~
```
Comment (crishpeen) [inline src/data/events.yml:1007]: company_link is not required, you can skip it.

**Review by crishpeen — APPROVED** [review]

File: src/data/events.yml (hunk 1 above)
Comment (adamkudrna) [reply src/data/events.yml:1007]: … or leave it `null` (`~`). It has the same effect, plus you can always see all the available options when copying the last event 🙂.

File: src/data/events.yml [hunk 2]
```diff
@@ -986,3 +986,34 @@
...
+            title: Metriky rychlosti načítání a&nbsp;tipy pro WebpageTest.org
+            youtube_id: fR9n_yi050g
+            image: martin-michalek.jpg
```
Comment (adamkudrna) [inline src/data/events.yml]: Could you please add links to Slideshare presentations of Martin and Láďa (see #27)? We don't have Iakov yet but we can add his slides later.

**Review by adamkudrna — CHANGES_REQUESTED** [review]

File: src/data/events.yml [hunk 3]
```diff
@@ -986,3 +986,36 @@
...
+    description: >
+        Středa 14.&nbsp;3. 2018, Ataccama office Praha
+    speakers:
+        -   name: Iakov Markov
+            name_link: https://twitter.com/IakovMarkov
+            company: Ataccama
+            company_link: https://www.ataccama.com/
+            title: Chrome&nbsp;DevTools, základní i&nbsp;pokročilé měření výkonu
+            youtube_id: 8DCQAJWwhFE
+            image: iakov-markov.jpg
```
Comment (adamkudrna) [inline src/data/events.yml]: … and here are Iakov's slides: https://docs.google.com/presentation/d/1b_V9H5vMOb2SkXh4Ev53hEADMlkM6FUL7HQriVtpbXI/edit#slide=id.p

**Review by adamkudrna — APPROVED** [review]

**Pull request merged**